as-controller-board-status/functions/output.go

```diff
@@ -262,16 +260,11 @@ func (boardStatus *CheckBoardStatus) processVendingDoorState(lc logger.LoggingCl
 	return nil
 }
 
-// GetStatus is a REST API endpoint that enables a web UI or some other downstream
-// service to inquire about the status of the upstream Automated Checkout hardware interface(s).
-func GetStatus(writer http.ResponseWriter, req *http.Request) {
-	utilities.ProcessCORS(writer, req, func(writer http.ResponseWriter, req *http.Request) {
-		controllerBoardStatusJSON, err := utilities.GetAsJSON(controllerBoardStatus)
-		if err != nil {
-			utilities.WriteStringHTTPResponse(writer, req, http.StatusInternalServerError, "Failed to serialize the controller board's current state.", true)
-			return
-		}
+func (boardStatus *CheckBoardStatus) GetControllerBoardStatus() ControllerBoardStatus {
+	return controllerBoardStatus
+}
 
-		utilities.WriteJSONHTTPResponse(writer, req, http.StatusOK, controllerBoardStatusJSON, false)
-	})
+// for testing purpose
+func (boardStatus *CheckBoardStatus) SetControllerBoardStatus(controllerBoardStatusInput ControllerBoardStatus) {
+	controllerBoardStatus = controllerBoardStatusInput
 }
```

Comment:
same comment above

Reply:
not a global variable

Reply:
unit test is the same package as output.go, on the same package so it is global to unit tests.

Reply:
> unit test is the same package as output.go, on the same package so it is global to unit tests.

@jim-wang-intel, If it is moved into the struct then it resolves the "global" issue. :-)

Reply:
issue #46 created to resolve this "global" issue

Reply:
yes, agreed, just discussed with @vli11 and we agreed to create a separate PR to resolve this.